recipes/asyncify/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "asyncify" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/a3/c6/33d42a1920395c9cfe912d4b6425504237b6bba47a48967aaf453b2b0d79/asyncify-0.2.2.tar.gz
+  sha256: 99cf0730feab2ee3600427336bf54dcf38ffa665436b91acf60f7c5ce6ad57a5
+
+build:
+  skip: true
```

Comment:
Please remove the ``skip: true`` otherwise it will skip all builds
```suggestion
```

Reply:
DONEZO